collections/_extensions/helm-plugin-for-meshery-snapshot.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: item
+name: Helm Plugin for Meshery Snapshot 
+kind: Plugin
+userName: Layer5, Inc.
+userAvatarURL: https://png.pngtree.com/png-vector/20191104/ourmid/pngtree-businessman-avatar-cartoon-style-png-image_1953664.jpg
+type: Collaboration
+compatibility: 
+  - aws
+  - kubernetes
+extensionId: 2d54f372-10e5-4c4e-8d23-b9c35c68ce98
+logo: /assets/images/meshery-logo.svg
+image: /assets/images/helm-dark.svg
+extensionInfo: |
+  Helm Plugin for Meshery Snapshots allow users to import packaged Helm charts as Meshery Designs and then Meshery Snapshots generate an image as a visual snapshot of Meshery designs.
+extensionCaveats: |
+  - A native helm CLI plugin for conveniently creating a visual snapshot of a packaged and unpackaged Helm chart
+  - Support the tradeoff between synchronous and asynchronous receipt of the snapshot
+  - Get snapshots of your infrastructure directly in your PRs
```

Comment:
There are better descriptors that can be found in the helm-kanvas-snapshot repo...

Reply:
@Aviral0702 are you incorporating this feedback?

Reply:
I'm not having much luck in my communication with @Aviral0702 either. I propose that one of you move forward with changes as needed.